diff --git a/minimatch.js b/minimatch.js
index 2e4a058a130dceebf83a7bd435f5d22b765ac836..b86f2c2e5214d778d7b2782862f47aa8b005f42d 100644
--- a/minimatch.js
+++ b/minimatch.js
@@ -8,6 +8,9 @@ minimatch.sep = path.sep
 
 var GLOBSTAR = minimatch.GLOBSTAR = Minimatch.GLOBSTAR = {}
 var expand = require('brace-expansion')
+if (expand && typeof expand !== 'function' && typeof expand.expand === 'function') {
+  expand = expand.expand
+}
 
 var plTypes = {
   '!': { open: '(?:(?!(?:', close: '))[^/]*?)'},
